### PR TITLE
Seb/refactor

### LINKED ISSUE #29: Discord Community Dropdown
- [ ] In the param have it include which discord community this is for
- [ ] Have it display this on the frontend
- [ ] Have a metameta contract that stores all the contract mappings per community
   - [ ] Include the admin per community, the server URL or identifier (only BAF Wallet Admins can update the metameta contract)

### PR DESCRIPTION
**Pull request by Sladuca**
- remove circular dependency between `libs/near` and `libs/community-contract`
- fix #29 - re-init app without refresh
- fix broken imports
- remove unnecessary imports 
- move `init.svelte` to `state` from `config`
- rename some functions so what they do is more clear
- moved UI components out of `libs/near` - they shouldn't live there.
- Addresses part of #33

### CODE REVIEW

File: apps/frontend/src/components/chains/ChainSettings.svelte [hunk 1]
```diff
@@ -2,39 +2,39 @@
...
+  import { chainStores, checkChainInit } from '../../state/chains.svelte';
+  import { initAppState } from '../../state/init.svelte';
   import { apiClient } from '../../config/api';
-  import { SiteKeyStore } from '../../state/keys.svelte';
-  import { AccountStore } from '../../state/accounts.svelte';
+  import { siteKeyStore } from '../../state/keys.svelte';
+  import { accountStore } from '../../state/accounts.svelte';
 
   const chains = getEnumValues(Chain);
   // TODO: clean up imports, see https://github.com/bafnetwork/baf-wallet-v2/issues/54
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/chains/ChainSettings.svelte:12]: remove the issue above

File: apps/frontend/src/components/chains/ChainSettings.svelte [hunk 2]
```diff
@@ -2,39 +2,39 @@
...
 
   const ChainInitAccountComponent = (chain: Chain) => () =>
-    import(`../../../../../libs/${chain}/src/web/InitAccount.svelte`);
+    import(`./${chain}/InitAccount.svelte`);
 </script>
 
 {#each chains as chain}
-  {#if checkChainInit($ChainStores, chain)}
+  {#if checkChainInit($chainStores, chain)}
     <!-- content here -->
```
Comment (Lev-Stambler) [inline apps/frontend/src/components/chains/ChainSettings.svelte:22]: remove comment (ik ik it was there from before)

File: apps/frontend/src/state/history.svelte [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
         data: string;
         sent: Date;
         confirmed: Date;
     }
-
     // TODO: somehow fetch tx history
-
     export const historyStore = writable<TxHistoryItem[]>([]);
-</script>
+</script>
```
Comment (Lev-Stambler) [inline apps/frontend/src/state/history.svelte:11]: new line

File: libs/near/src/lib/tx.ts [hunk 4]
```diff
@@ -51,7 +50,10 @@ export interface NearBuildTxParams {
   senderAccountID: NearAccountID;
   recipientAccountID: NearAccountID;
 }
-export interface NearSignTxOpts {}
+
+export interface NearSignTxOpts {
+  _: string;
```
Comment (Lev-Stambler) [inline libs/near/src/lib/tx.ts:55]: what does the `_` do? Could we comment this above?

File: libs/near/src/lib/rpc.ts [hunk 5]
```diff
@@ -12,7 +12,10 @@ export type NearRpcInterface = RpcInterface<
 
 // TODO: go spelunking in near's jsonRpcProvider and see what options it takes
 // and see what kind of stuff it actually returns so we can do better than 'any'
-export interface NearSendOpts {}
+export interface NearSendOpts {
```
Comment (Lev-Stambler) [inline libs/near/src/lib/rpc.ts:15]: what does the `_` do? Could we comment this above?

**Review by Lev-Stambler — COMMENTED** [review]
Looks great!! Just a few things.

Also, I was wondering how this fixes the reinit without refreshing?

File: libs/near/src/lib/tx.ts (hunk 4 above)
Comment (Sladuca) [reply libs/near/src/lib/tx.ts:55]: oh yeah, that was to get eslint to stop complaining

**Review by Lev-Stambler — APPROVED** [review]
Looks gtm.

**Pull request merged**